right/src/slave_drivers/uhk_module_driver.c

```diff
@@ -73,6 +75,25 @@ void UhkModuleSlaveDriver_Init(uint8_t uhkModuleDriverId)
     uhkModuleState->pointerDelta.y = 0;
 }
 
+// When module is swapped, we need to reload its Keymap once we know its
+// moduleId. However, this triggers macro events, and we don't want to trigger
+// the same macro multiple times if we can detect that another module is also
+// going through the initialization sequence.
+void ReloadKeymapIfNeeded()
```

Comment:
Should be lowerCamelCase because it's a local function.

Reply:
Indeed.